src/main.rs

```diff
@@ -64,37 +70,38 @@ async fn main() {
 #[tracing::instrument(level = "trace")]
 async fn cmd_install(args: InstallArgs, options: Options) {
     let packages = args.packages;
-    let sorted = sort(&packages, options).await;
 
-    if !sorted.repo.is_empty() {
-        operations::install(sorted.repo, options).await;
-    }
-    if !sorted.aur.is_empty() {
-        operations::aur_install(sorted.aur, options).await;
-    }
-    if !sorted.nf.is_empty() {
-        crash!(
-            AppExitCode::PacmanError,
-            "Couldn't find packages: {} in repos or the AUR",
-            sorted.nf.join(", ")
-        );
+    let arg1 = env::args().collect::<Vec<String>>()[1].clone();
```

Comment:
Why are we collecting the arguments?

Reply:
Ah, I just discovered `env::args().nth()` exists, I'll change this

Reply:
No I mean can clap handle it?

Reply:
As far as I'm aware there's no way to see which Alias was passed in, only what subcommand the alias evaluates to

Reply:
I'll remove this extra logic for now and open it as a separate PR, fixing searches whatsoever takes priority

Reply:
https://docs.rs/clap/3.2.20/clap/builder/struct.App.html#method.short_flag

I found EXACTLY what we're looking for.

Reply:
One issue with this is that we wouldn't be able to handle `-Ss` in this way as `-S` would be the `short_flag` of `install`

Reply:
You can mix derive and Builder pattern if you need to

Reply:
Would this and the other comment be fine marked as resolved for now as I will reimplement these in another PR?

Reply:
Yep